redis/pool.go

```diff
@@ -299,34 +299,61 @@ func (p *Pool) lazyInit() {
 	p.mu.Unlock()
 }
 
-// get prunes stale connections and returns a connection from the idle list or
-// creates a new connection.
-func (p *Pool) get(ctx context.Context) (*poolConn, error) {
+// waitVacantConn waits while at least one vacant connection appears in the pool.
```

Comment:
This doesn't reflect what the code actually does.

Reply:
How about `waitVacantConn waits until at least one vacant connection gets spotted in the pool.` ?